node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,8 @@
+import { readFile } from 'node:fs/promises';
+try {
+  const filePath = new URL(String(process.argv[2]), import.meta.url);
```

Comment:
`String(process.argv[2])` is not necessary since `process.argv[2]` is already a string, so you can just use `process.argv[2]` directly.